lib/Layout/LayoutOverlaySidebar.js

```diff
@@ -4,7 +4,7 @@ import * as easings from 'd3-ease';
 import cx from 'classnames';
 import { useTransition, animated } from 'react-spring';
 
-export function Sidebar({
+export function LayoutOverlaySidebar({
```

Comment:
The existing `Sidebar` is only ever used in `Layout` so I moved it next to that area to make it easier to understand and to separate the concerns of the contents of a sidebar and the layout aspect of it.